contracts/staking/StakingInfo.sol

```diff
@@ -466,9 +467,10 @@ contract StakingInfo is Ownable {
         uint256 validatorId,
         address user,
         uint256 amount,
-        uint256 tokens
+        uint256 tokens,
+        uint256 burnId
     ) public onlyValidatorContract(validatorId) {
-        emit ShareBurned(validatorId, user, amount, tokens);
+        emit ShareBurned(validatorId, user, amount, tokens, burnId);
```

Comment:
Should we also change the name of the event? Otherwise indexing it would be hard. 
Something like `SharedBurnedWithBurnId`?

Reply:
I am not sure, if it is better to have a separate event, I will do that.

Reply:
On bad thing, I realized that we don't have a proxy in stakingInfo!